frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -39,6 +41,8 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
     catalogSourceNamespace,
   } = item;
   const notAvailable = <span className="properties-side-panel-pf-property-label">N/A</span>;
+  const actionText =
+    _.get(obj, ['metadata', 'annotations', 'marketplace.openshift.io/action-text']) || 'Install';
```

Comment:
I'm guessing these annotations would be on the CSV not the PackageManifest, but we need to check with @alimobrem. If so, this would be

```suggestion
    _.get(obj, ['status', 'channels', '0', 'currentCSVDesc', 'annotations', 'marketplace.openshift.io/action-text']) || 'Install';
```

and similar for the annotations below

Reply:
Yes they are on the CSV.

Thanks